auxprogs/compileSpliceCands/Makefile

```
@@ -1,8 +1,9 @@
 compileSpliceCands : compileSpliceCands.o list.h list.o
-	gcc $(LDFLAGS) -o compileSpliceCands compileSpliceCands.o list.o;
+	$(CC) -o compileSpliceCands compileSpliceCands.o list.o $(CFLAGS) $(LDFLAGS);
 #	cp compileSpliceCands ../../bin
+
 compileSpliceCands.o : compileSpliceCands.c 
-	gcc -Wall -pedantic -ansi $(CPPFLAGS) -c compileSpliceCands.c 
+	$(CC) -Wall -pedantic -ansi $(CFLAGS) $(CPPFLAGS) -c compileSpliceCands.c
```

```suggestion
	$(CC) -Wall -pedantic -ansi $(CFLAGS) $(CPPFLAGS) $(LDFLAGS) -c compileSpliceCands.c
```
- $(LDFLAGS) missing?

---

not required